react-intl/index.d.ts

```diff
@@ -34,14 +34,16 @@ declare namespace ReactIntl {
         formats?: any;
     }
 
-    interface InjectedIntl {
+    interface InjectedIntlProps {
+      intl: {
```

Comment:
I tried, but I cant find a better solution than making the prop "intl" nullable.